tests/e2e/setup/build-and-push-operator.sh

```diff
@@ -86,7 +97,13 @@ build_and_push_operator_image() {
 
 # Main logic
 if [ "${OCP}" == "true" ]; then
-  get_internal_registry
+  if [ "${CI}" == "false" ]; then
```

Comment:
I'm not sure if CI is a good name, here it only controls if we are going to skip internal registry config which must happen later anyway no matter if it's CI run or localhost run.